app/index.html

```diff
@@ -15,6 +15,16 @@
       </strong>
     </noscript>
     <div id="app"></div>
+    <script>
+      window.global = window;
+    </script>
+    <script type="module">
+      import process from "process";
+      window.process = process;
+    </script>
+    <script>
+      let global = globalThis;
```

Comment:
where is `globalThis` being populated from ?
would it make sense to combine L19 & L26 under the same script tag ?

Reply:
`globalThis` exists on every web page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis, and since browsers, unlike node.js, don't have a default `global` instance, we need to define it here since some npm packages rely on `global`

Also in favor of combing those script tags for brevity